cli/repo/repo_update.go

```diff
@@ -87,8 +92,30 @@ func repoUpdate(ctx context.Context, c *cli.Command) error {
 	if c.IsSet("trusted") {
 		patch.IsTrusted = &trusted
 	}
+	// TODO: remove isGated in next major release
 	if c.IsSet("gated") {
-		patch.IsGated = &gated
+		log.Warn().Msgf("Deprecated 'gated' option was used, use 'require-approval' in the future to update an repo settings.")
```

Comment:
same: 

might have a better message or is this ok?

Reply:
```suggestion
		log.Warn().Msgf("The 'gated' flag was deprecated, use 'require-approval' instead.")
```

Reply:
b142c677d21ac4192542c9590326412defdb14ba